docker-compose.yaml

```diff
@@ -14,9 +14,9 @@ services:
     volumes:
       - ./data/postgres:/var/lib/postgresql/data
     environment:
-      - POSTGRES_DB=cantus_db
-      - POSTGRES_USER=cantus_admin
-      - POSTGRES_PASSWORD=Pl4c3H0ld3r
+      - POSTGRES_DB=${POSTGRES_DB}
```

Comment:
This only works iff there is an `.env` file, otherwise you set an empty password, right? Kind of important. Not sure if there's a better way to do this.

Reply:
I did not test how this requirement makes itself known, but the docs say that it cannot be empty, and that this environment variable is required by the container. So, I think for deployment anywhere in Canada we should require that this be set and not left empty? Let me see if I can check how this is done -- whether it errors in an informative way, etc?

Reply:
OK - so if you don't specify the password, the container build fails with:
<img width="682" alt="image" src="https://user-images.githubusercontent.com/11023634/171045308-aa79a5d9-0b70-4f72-9e3a-29d3b78e4be0.png">

I do agree that we want to force people to set a password and that we don't want to have a default password written in the codebase somewhere that could go to production without being modified or reviewed.